src/server/common/utilities/types.ts

```diff
@@ -0,0 +1,7 @@
+export const isObjectRecord
+= (value: unknown): value is Record<string, unknown> => (
+  typeof value === 'object'
+  && value !== null
+  && !Array.isArray(value)
+);
+
```

Comment:
Rather than having `src/client/common/utilities/types.ts` and `src/server/common/utilities/types.ts`, we can consolidate these into one file that gets imported from both the server and the client, located at `src/common/utilities/types.ts`.